src/osd/SnapMapper.cc

```diff
@@ -40,7 +40,6 @@ using result_t = Scrub::SnapMapReaderI::result_t;
 using code_t = Scrub::SnapMapReaderI::result_t::code_t;
 
 
-const string SnapMapper::LEGACY_MAPPING_PREFIX = "MAP_";
```

Comment:
Before removing `MAP_`, `removed_snap_`,`removed_epoch_` I think that we should note somewhere that these keys were used before octopus in case someone tries to understand where they came from.

Reply:
Makes sense.